Source/AwsCommonRuntimeKit/http/HttpRequest.swift

```diff
@@ -3,49 +3,69 @@
 import AwsCHttp
 import AwsCIo
 import AwsCCommon
-// TODO: HttpRequest & HttpMessage relationship
+
 public class HttpRequest: HttpMessage {
 
-    public override init(allocator: Allocator = defaultAllocator) throws {
-        try super.init(allocator: allocator)
+    public init(method: String = "GET",
+                path: String = "/",
+                headers: HttpHeaders? = nil,
+                body: IStreamable? = nil,
+                allocator: Allocator = defaultAllocator) throws {
```

Comment:
you could remove `throws` from this

`aws_http_message_set_request_method()` can't *really* fail
`aws_http_message_set_request_path()` can't *really* fail

these return `int` because they were written in the bad old days when allocations could fail

Reply:
Thanks, fixed.